### PR TITLE
Ear 336 history cancel button

### PR DESCRIPTION
**Pull request by farres1**
> BEFORE MAKING YOUR PR... There must be no linting errors and all tests must pass

### What is the context of this PR?

Implementing a cancel button to cancel an edit being made to a history item.

### How to review 

> Add to the list below as appropriate, including screenshots when necessary

* Import the **Capability examples** questionnaire from pre-prod Author into your local environment and:
    * ensure it can be opened in Author;
    * then, ensure it can be viewed in Runner by pressing the **view survey** button
* Does this require a migration? We need one if existing JSON schema properties change

### What to do after everything is green

1. * [ ] Bring this branch up-to-date with Origin/Master
2. * [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. * [ ] Click the **Merge** button on this pull request
4. * [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. * [ ] Move the Jira ticket for this task into the next stage of the process

### CODE REVIEW

**Review by jasehumphr — DISMISSED** [review]

File: eq-author/src/App/history/HistoryItem.js [hunk 1]
```diff
@@ -112,6 +112,12 @@ const SaveButton = styled(Button)`
   margin-bottom: 1em;
 `;
 
+const CancelButton = styled(Button)`
```
Comment (tjamesmac) [inline eq-author/src/App/history/HistoryItem.js:115]:
```suggestion
const ActionButton = styled(Button)`
```

File: eq-author/src/App/history/HistoryItem.js [hunk 2]
```diff
@@ -162,6 +168,15 @@ const HistoryItem = ({
...
+            <CancelButton
+              data-test="cancel-note-btn"
+              onClick={() => {
+                setNoteState(bodyText);
+                setIsEditActive(false);
+              }}
+            >
+              Cancel
+            </CancelButton>
             <SaveButton
```
Comment (tjamesmac) [inline eq-author/src/App/history/HistoryItem.js:180]:
```suggestion
            <ActionButton
              data-test="cancel-note-btn"
              onClick={() => {
                setNoteState(bodyText);
                setIsEditActive(false);
              }}
            >
              Cancel
            </ActionButton>
            <ActionButton
```

**Review by tjamesmac — APPROVED** [review]
Hey @farres1, looks really good! Only one suggestion and it would be to merge the cancel/save button into one component and then use that one twice

File: eq-author/src/App/history/HistoryItem.js (hunk 2 above)
Comment (jasehumphr) [reply eq-author/src/App/history/HistoryItem.js:180]: Yeah that's a good idea

**Comment by farres1** [comment]
> Hey @farres1, looks really good! Only one suggestion and it would be to merge the cancel/save button into one component and then use that one twice

I forgot to do this, I'll change it now. Thank you!

**Comment by tjamesmac** [comment]
Okay @farres1 that's much better but I have two more things now.

Could you align the buttons so the right edge of the save button is inline with the text box? 

Also the cancel button currently has the wrong styling (checkout the `calSumContentPicker` cancel button)

Then it's good to go!

**Review by jasehumphr — APPROVED** [review]
Looks great Sam

**Review by tjamesmac — COMMENTED** [review]
Looks awesome Sam, thanks for making those changes

**Review by tjamesmac — APPROVED** [review]

**Pull request merged**